src/pages/explore/index.js

```diff
@@ -72,14 +75,15 @@ Explore.defaultProps = {
   pageSize: undefined,
   query: undefined,
   theme: undefined,
+  ordering: undefined,
   tweets: undefined,
   results: undefined,
 };
 
 export async function getServerSideProps(context) {
   let results = null;
   const { query: userQuery } = context;
-  const query = { days: 14, ...userQuery };
+  const query = { days: 14, ordering: "deleted_at", ...userQuery };
```

Comment:
```suggestion
  const query = { days: 14, ordering: "-deleted_at", ...userQuery };
```

Most recently deleted tweet should be first by default.